sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_scalar_aggregates_v1.sql.py

```diff
@@ -96,7 +96,8 @@ def generate_sql(
                 app_version,
                 app_build_id,
                 channel,
-                {aggregates}
+                {aggregates},
+                max(profile_group_id) AS profile_group_id
```

Comment:
Can the value of `profile_group_id` change? If yes, using `MAX()` could potentially leave us with the profile_group_id evolving over time for a specific client.

Reply:
Good question - profile group ID should not change over time - it should be once it is assigned it stays consistent.  I did debate changing the grain to include the grain by profile group ID instead, but I thought this could potentially break downstream things that expect the grain to be a certain level already.